src/backends/pgfplotsx.jl

```diff
@@ -240,6 +240,16 @@ function (pgfx_plot::PGFPlotsXPlot)(plt::Plot{PGFPlotsXBackend})
                 PGFPlotsX.Axis
             end
             axis = axisf(axis_opt)
+            if sp[:legendtitle] !== nothing
+                push!(axis, PGFPlotsX.Options("\\addlegendimage{empty legend}" => nothing))
```

Comment:
actually it should be enough to push the string here, but if it works with the `Options` you can also leave it as it is

Reply:
Oh, I was not aware of that, thanks for the info!